onnxruntime/core/graph/graph_transformer_mgr.h

```diff
@@ -4,14 +4,22 @@
 #pragma once
 
 #include "core/graph/graph_transformer.h"
+#include "core/graph/constant_folding.h"
 
 namespace onnxruntime {
 // Manages a list of graph transformers. It is initialized with a list of graph
 // transformers. Each inference session can further register additional ones.
 class GraphTransformerManager {
  public:
-  explicit GraphTransformerManager(unsigned steps) noexcept : steps_(steps) {
-    // TODO: Register default transformers.
+  explicit GraphTransformerManager(unsigned steps, bool enable_default_transformers) noexcept : steps_(steps) {
+    // Register default transformers.
+    if (enable_default_transformers) {
+      std::unique_ptr<TopDownRuleBasedTransformer> rule_transformer =
+          std::make_unique<TopDownRuleBasedTransformer>("DefaultRuleTransformer",
```

Comment:
As we add more rules, we might want to keep it open to add a bottom up traversal, as there might be some rules that are better applicable bottom up.

Reply:
Since there is no such scenario for now, I would change it to RuleBasedTransformer.  Actually, the loop in function TopDownRuleBasedTransformer::Apply can be changed to traverse graph.Nodes() rather than graph_viewer.GetNodesInTopologicalOrder().

Reply:
I removed the default transformers in the last commit. #556 will take care of which transformers will be enabled.
I kept the TopDownRuleBasedTransformer -- it makes people that are deeper in the code understand how their graph is traversed, while the API is agnostic of it.